docs/lokomotive/_index.md

```diff
@@ -3,10 +3,14 @@ content_type: lokomotive
 title: Lokomotive
 children_are_versions: true
 external_docs:
-  - repo: https://github.com/kinvolk/lokomotive.git
-    name: "0.5"
-    branch: "master"
-    dir: "docs"
+- repo: https://github.com/kinvolk/lokomotive.git
+  name: "0.6"
+  branch: "master"
+  dir: "docs"
+- repo: https://github.com/kinvolk/lokomotive.git
+  name: "0.5"
+  branch: "v0.5.0"
+  dir: "docs"
```

Comment:
I think this should work.

```suggestion
- repo: https://github.com/kinvolk/lokomotive.git
  name: "0.6"
  branch: "v0.6.0"
  dir: "docs"
- repo: https://github.com/kinvolk/lokomotive.git
  name: "current"
  branch: "master"
  dir: "docs"
```

Reply:
Wouldn't removing `0.5.0` tag unlinks all the URLs with 0.5.0 in them? Or there is a provision to redirect such URLs?

Reply:
That's a good point. I think we can keep `0.5` and point it to the first commit with the new docs (which is not going to be `v0.5.0` but is as closest as possible) to keep the current `0.5` links.

Reply:
Filed https://github.com/kinvolk/docs/pull/11